plugins/yaml-schema/CHANGELOG.md

```diff
@@ -0,0 +1,5 @@
+# @flatfile/plugin-yaml-schema
+
+## 0.0.1
+
+### Patch Changes
```

Comment:
Run `npm run changeset`, update as a patch and add a release description (i.e. "Introducting the @flatfile/plugin-convert-yaml-schema plugin to configure a Flatfile Space based on a provided YAML."